client/service/src/builder.rs

```diff
@@ -523,6 +518,11 @@ impl<TBl, TRtApi, TCl, TFchr, TSc, TImpQu, TFprb, TFpp, TExPool, TRpc, Backend>
 		self.remote_backend.clone()
 	}
 
+	/// Consume the builder and return the parts needed for chain operations.
+	pub fn to_chain_ops_parts(self) -> (Arc<TCl>, Arc<Backend>, TImpQu, TaskManager) {
```

Comment:
Why add a special method for this?

What happens if I require more next week? Maybe the tx pool? Can we not just call build add take what we need?

Reply:
The import queue gets consumed in build, so we need to get it out before that.

Reply:
This method is only temporary, and will only be here until I remove the service builder.

Reply:
Okay :+1: